source/adressering-av-undertegner.rst

```diff
@@ -1,78 +1,75 @@
-.. _adressering-av-undertegner:
+.. _addressing-of-signer:
 
-Adressering av undertegner
+Addressing of signer
 ***************************
-Undertegnere som skal signere i :ref:`portalflyt <signering-i-portalflyt>` kan adresseres på to forskjellige måter;
+Signers that are to sign in :ref:`portal flow <signing-in-portal-flow>` may be addressed in two different ways:
 
-1. På e-post og/eller SMS (Kun tilgjengelig for private virksomheter)
-2. Med fødselsnummer
+1. By email and/or SMS (only available for private organizations)
+2. By national identity number
 
-..  CAUTION::
-    Måten du adresserer undertegner på, påvirker :ref:`innhold og utseendet på undertegners signatur <identifisereUndertegnere>`.
+..  WARNING::
+    How you address a signer affects the :ref:`content and appearance of the signer's signature <identify-signers>`.
 
-1. Adressering på e-post / SMS
+1. Address using e-mail / SMS
 ===============================
 
 .. NOTE::
-   Dette alternativet er kun tilgjengelig for private virksomheter
+   This option is only available for private organizations
 
-Ved adressering på e-post / SMS knyttes ikke signeringsforespørselen til et fødselsnummer. Vi har derfor ikke noen måte å verifisere at det er riktig person som leser og signerer. Det kreves derfor ikke innlogging for å lese og signere. Mottaker får e-post og/eller SMS med en unik kode og lenke som gir tilgang til forespørselen. Se :ref:`varsler <varslerUtenFødselsnummer>` for å se hvordan varselet ser ut.
+On addressing by email/SMS, the signature request is not linked to a national identity number. We therefore do not have any way of verifying that the correct person is reading and signing. It is therefore not necessary to log in so as to read and sign. The recipient receives an email and/or SMS with a unique code and link to access the request. View :ref:`notifications <notifications-without-national-identity>` to see what the notification looks like.
```

Comment:
```suggestion
When addressing signers by email/SMS, we will not be able to link signers to their national identity number. Therefore, we do not have any way to verify that the correct person is reading and signing. The recipients receive an email and/or SMS with a unique code and link to access the document(s) to sign, instead of being required to log in. See :ref:`notifications <notifications-without-national-identity>` to review what the notifications look like.
```